### PR TITLE
add 2025 PV USA capacity for upper bound

### PR DESCRIPTION
**Pull request by robertpietzcker**

### CODE REVIEW

File: .zenodo.json
Comment (fbenke-pik) [inline .zenodo.json:1]: did you delete this by accident, or was it automatically removed while running buildLibrary()

Comment (fbenke-pik) [reply .zenodo.json:1]: looks like this was replaced by CITATION.cff as part of the build routine!? should be fine, i guess

**Review by fbenke-pik — APPROVED** [review]
Looks good to me

File: .zenodo.json
Comment (robertpietzcker) [reply .zenodo.json:1]: accidentally removed - I have no idea about this. so if you say it is fine, I am happy :-)

the only thing I changed on purpose was the calcCapacity part, and there I simply copied Chris addition to CHA and changed it for the US

**Pull request merged**